apps/registration-api/regAuthCheck.js

```diff
@@ -1,21 +1,23 @@
 import fetch from 'node-fetch'
-const authURL = process.env.authURL
+const authURL = process.env.authURL || 'https://auth.sagecontinuum.org'
```

Comment:
This is fine.  In the future, I'd suggest putting defaults in a config file or such.